tests/bsim/bluetooth/audio/src/bap_broadcast_assistant_test.c

```diff
@@ -447,7 +486,7 @@ static void test_main(void)
 	err = bt_le_scan_start(BT_LE_SCAN_PASSIVE, device_found);
 	if (err != 0) {
 		FAIL("Scanning failed to start (err %d)\n", err);
-		return;
+		return err;
```

Comment:
```suggestion

		return err;
```

Reply:
See https://github.com/zephyrproject-rtos/zephyr/pull/51679#discussion_r1156943149

We could modify it, but that would be a separate change for the entire file. Let me know if you want me to update that

Reply:
This is really a nitpick, I think this can be as it is.